app/views/pages/privacy_policy.html.slim

```diff
@@ -1,8 +1,43 @@
 - title t('titles.privacy_policy')
 
-h1.h3.mt0.mb1 = t('headings.privacy_policy.security_consent')
-p.mb4 = t('pages.privacy_policy.security_consent')
-h1.h3.mt0.mb1 = t('headings.privacy_policy.privacy_act')
-p.mb4 = t('pages.privacy_policy.privacy_act')
-= link_to t('links.create_account'), new_user_registration_path,
-  class: 'btn btn-primary btn-wide mb1'
+p = t('pages.privacy_policy.by_using')
+
+h1.h3.mt0.mb1 = t('headings.privacy_policy.security_practices')
+
+p = t('pages.privacy_policy.industry_standards')
+
+p = t('pages.privacy_policy.unauthorized_access')
+
+h1.h3.mt0.mb1 = t('headings.privacy_policy.privacy_practices')
+
+p = t('pages.privacy_policy.privacy_notice')
+
+p = t('pages.privacy_policy.privacy_goal')
+
+h2.h4.mt0.mb2 = t('headings.privacy_policy.privacy_act')
+
+h3.h5.mt0.mb1 = t('headings.privacy_policy.authorities')
+
+p = t('pages.privacy_policy.e_government_text_html', \
+      e_government_link: link_to( \
+        t('pages.privacy_policy.e_government_link'), \
+        'https://www.gpo.gov/fdsys/pkg/PLAW-107publ347/html/PLAW-107publ347.htm'))
+
+h3.h5.mt0.mb1 = t('headings.privacy_policy.purpose')
+
+p = t('pages.privacy_policy.use_of_information')
+
+h3.h5.mt0.mb1 = t('headings.privacy_policy.disclosure')
+
+p = t('pages.privacy_policy.you_decide')
+
+h3.h5.mt0.mb1 = t('headings.privacy_policy.routine_uses')
+
+p = t('pages.privacy_policy.applicable_federal_agency_html', \
+      systems_of_record_notices_link: link_to( \
+        t('pages.privacy_policy.systems_of_record_notices'), '#'))
```

Comment:
we don't have the link at the moment, I've created an issue to track adding in the real link